### PR TITLE
Vanity tracking

### PR DESCRIPTION
**Pull request by seanmtracey**
Now, when we generate a vanity URL, the url shown will first go through our service and then be forwarded on to our corporate partners landing page. Also, the checking of valid partners has been modularised. But, wait... there's more! You can now also display Big FT in black and white with ?monotone=true.

### CODE REVIEW

File: client/src/js/queryParams.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ const edition = (parsed.edition) ? parsed.edition : defaultEdition;
 const organisation = parsed.organisation;
 
 const partner = parsed.partner;
+const monotone = parsed.monotone === "true" ? true : false;
```
Comment (JakeChampion) [inline client/src/js/queryParams.js]: `parsed.monotone === "true"` evaluates to a boolean. No need for the ternary.

File: client/src/main.js [hunk 2]
```diff
@@ -435,7 +436,7 @@ const __bigFT = (function (){
 					const data = json.data;
 
 					if(data.url !== undefined && data.url !== ''){
-						partnerURLElement.textContent = data.url;
+						partnerURLElement.textContent = `${window.location.host}/${data.partner}`;
```
Comment (JakeChampion) [inline client/src/main.js]: What's this about?

File: client/src/main.scss [hunk 3]
```diff
@@ -17,6 +17,11 @@ body {
 	font-size: 5vh;
 }
 
+.monotone{
+	-webkit-filter: grayscale(100%);
```
Comment (JakeChampion) [inline client/src/main.scss]: OBT comes with autoprefixer so you don't have to worry about writing browser prefixes in your sass

Comment (seanmtracey) [inline client/src/main.scss]: Sweet. OBT FTW!

File: client/src/js/queryParams.js (hunk 1 above)
Comment (seanmtracey) [inline client/src/js/queryParams.js]: Yup, my bad.

**Pull request merged**